### DISCUSSION TITLE
Why this repository moved to new address

### DISCUSSION THREAD

**Discussion by yartasdev** [ngxpert/hot-toast#3] [General]
This was the first thing I noticed while updating the dependencies in my own project, why was this repository moved to a different address?

**Comment by shhdharmen** [comment]
https://twitter.com/NetanelBasal/status/1762512595435528531?t=NJcXDw-RjQ42m90nI5U1ZA&s=19

**Comment by DaSchTour** [comment]
This still gives me headaches how this was suddenly "moved"
